lib/registration.dart

```diff
@@ -326,16 +328,44 @@ class _RegistrationState extends State<Registration> {
                                     ),
                                   ),
                                   haveCar == true
-                                      ? Container(
-                                          width: width * 0.85,
-                                          child: CarPlateForm(
-                                            onChanged: (plate) {
-                                              carPlate = plate;
-                                            },
-                                          ),
-                                        )
+                                      ? ListView.separated(
+                                          separatorBuilder: (context, index) =>
+                                              Divider(
+                                                color: Colors.white,
+                                              ),
+                                          shrinkWrap: true,
+                                          itemCount: carPlates.length,
+                                          itemBuilder: (BuildContext context,
+                                              int index) {
+                                            return Dismissible(
```

Comment:
I don't think you need `Dismissable` here, can we remove it ?

Reply:
Rremoved